tests/TestCase.php

```diff
@@ -30,21 +32,16 @@ protected function getPackageProviders($app)
 
     protected function setNow($year, int $month = 1, int $day = 1)
     {
-        $newNow = $year instanceof Carbon
-            ? $year
+        $newNow = ($year instanceof Carbon || $year instanceof CarbonInterface)
```

Comment:
Do we still need to check for `Carbon` if we already check for `CarbonInterface`?

Reply:
If we want to support Carbon 1, yes